mutatex/utils.py

```diff
@@ -435,12 +439,25 @@ def get_foldx_sequence(pdb, multimers=True):
                     residue_list.append(tuple(["%s%s%d" % (res_code, chain.get_id(), residue.get_id()[1])]))
                 else:
                     sequences[chain_name] += res_code
+                    positions[chain_name] += f"{residue.get_id()[1]},"
 
     if multimers:
         collated_chains = []
         seq_ids, seqs = list(zip(*list(iteritems(sequences))))
+        pos_ids, pos = list(zip(*list(iteritems(positions))))
         seq_ids = np.array(seq_ids)
+        pos_ids = np.array(pos_ids)
+        
         unique_seqs, unique_idxs = np.unique(seqs, return_inverse=True)
+        unique_pos, unique_idxp = np.unique(pos, return_inverse=True)
+        
+        try:
+            if not (unique_idxs == unique_idxp).all():
+                log.warning("Input amino acid sequence and input position sequence is not identical in multimer.")
+                raise ValueError
+        except ValueError as e:
+            log.error("The supplied PDB files must have identical positions of sequences. Exiting...")
+            exit(1)
```

Comment:
the idea behind try/except clauses is that you "try" to do something and then handle if it fails. In a case like this in which we `raise` an exception explicitly, it should then be handled in the main script (which is also where also all the rest of the `exit()` statements in case of problems is). If you `raise` and `except` right after, it works mostly in the same way as a `if`/`else` statement. Here we want to decouple error raise from error handling, so that downstream code can handle the exception as they please and the handling is not tied to the definition of the function. For instance:

So something like:

`mutatex`:
```
try:
    seq = get_foldx_sequence(...)
except ValueError as e:
    logo.error("{e.message}. Exiting...")
    exit(1)
```

`utils.py`:
```
        if not (...):
            raise ValueError("my custom error message")
```

in this way, if I import my function e.g. in another program or another script, it doesn't necessarily exit every time it has this problem but the error can be handled in different ways in a case-dependent manner. In general, `exit` statements are better suited for the `__main__` part of the code (so the `mutatex` script in this case), so that functions don't make any program that imports them exit